vulnerabilities/dos-revert.md

```diff
@@ -87,6 +87,15 @@ It's important to take caution in enforcing expected contract balances of tokens
 
 Consider, for example, a contract which expects the Ether balance to be 0 for the first deposit to allow for custom accounting logic. An attacker may forcibly send Ether to the contract before the first deposit, causing all deposits to revert. 
 
+### Divide by zero
+In solidity if the contract attempts to perform division when the denominator is ``zero``, the whole transaction reverts. Thus, the denominator should be always checked before division to prevent DoS revert.
```

Comment:
A bit pedantic but it would be more accurate to say that the call reverts because the whole transaction won't necessarily revert

Reply:
Makes sense. Fixed in the latest commit!